org.musicbrainz.Picard.appdata.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.musicbrainz.Picard</id>
+  <project_license>GPL-2.0+</project_license>
```

Comment:
@bochecha Hi :) https://spdx.org/licenses/GPL-2.0+ says it's deprecated. It doesn't say what it's deprecated in favour of, but https://spdx.org/licenses/ has "GPL v2.0 or later" as [`GPL-2.0-or-later`](https://spdx.org/licenses/GPL-2.0-or-later.html). Should the license be updated/changed here too?

Reply:
The problem is SPDX has multiple versions, and the appstream spec doesn't say which version it relies on.

However the examples use the deprecated form: https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-project_license

When in doubt, the best is to use `appstream-util validate-relax` to verify. Both seem to be accepted, so yes, let's move to the newer SPDX identifier. :slightly_smiling_face:

Reply:
@abhi-ohri :point_up_2: :)

Reply:
Let's got with the newer syntax "GPL-2.0-or-later" then

Reply:
```suggestion
  <project_license>GPL-2.0-or-later</project_license>
```
Done.